docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,33 +19,42 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ens.object.head(5)  # Grabs the first5 rows of the object table
</pre><p>nit: first5-&gt; first 5</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>